fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/CreateIndexStatement.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.common;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+import com.ibm.fhir.database.utils.api.IDatabaseStatement;
+import com.ibm.fhir.database.utils.api.IDatabaseTranslator;
+import com.ibm.fhir.database.utils.model.OrderedColumnDef;
+
+/**
+ * Create an index
+ */
+public class CreateIndexStatement implements IDatabaseStatement {
```

Comment:
How is this different than what we had before? The ordered columns?

Reply:
It's used in a different context...as part of the migration step Lee implemented. There's some redundancy here, but at the time I was fighting the Derby/Liberty alter table bug.